jekyll/_cci2/optimization-cookbook.md

```diff
@@ -0,0 +1,215 @@
+---
+layout: classic-docs
+title: "CircleCI Optimizations Cookbook"
+short-title: "Optimizations Cookbook"
+description: "Starting point for Optimizations Cookbook"
+categories: [getting-started]
+order: 1
+---
+
+The *CircleCI Optimizations Cookbook* is a collection of individual use cases (referred to as "recipes") that provide you with detailed, step-by-step instructions on how to perform various optimization tasks using CircleCI resources. This guide, and it associated sections, will enable you to quickly and easily perform repeatable optimization tasks on the CircleCI platform.
+
+* TOC
+{:toc}
+
+## Introduction
+
+Sometimes when you are using the CircleCI platform, you may encounter unexpected lags in pipeline performance, which can negatively affect your ability to perform critical organizational functions. These performance bottlenecks can not only impact overall performance, but also cause workflow and build failures. These "hiccups" can cost you money in terms of credit usage, resources, and individual time spent reducing bottlenecks.
+
+This guide provides you with the following optimization strategies that you can utilize to minimize any potential performance bottlenecks and ensure that you are getting the best performance possible when using CircleCI:
+
+- Running jobs sequentially to prevent concurrency
+- Implementing caching strategies to optimize builds and workflows
+- Improving test performance
+
+**Note:** This guide will be updated with new optimization strategies on a continual basis, so please feel free to refer to this page for new and updated content.
+
+### Optimization Recipes
+
+The table below lists the different optimization tasks that can be performed using the CircleCI platform.
+
+Recipe | Description 
```

Comment:
This table is basically the same info as the bullet list above? maybe each can link to the section headings below?

Reply:
great comment @rosieyohannan 👍 I went ahead and removed the table and added a TOC so the user can easily jump to the different sections on the page. Thanks :)